contract/src/ibcToken.js

```diff
@@ -0,0 +1,68 @@
+// @ts-check
+import { AssetKind, makeIssuerKit } from '@agoric/ertp';
+import { M, prepareExoClass } from '@agoric/vat-data';
+
+/**
+ * @typedef {{
+ *  tokenKeyword: string;
+ *  tokenDecimals: number;
+ *  proposedName: string;
+ *  denom: string;
+ * }} IBCTokenTerms
+ */
+
+export const meta = {
+  customTermsShape: {
+    tokenKeyword: M.string(),
+    tokenDecimals: M.number(),
+    proposedName: M.string(),
+    denom: M.string(),
+  },
+};
+harden(meta);
+
+export const IBCTokenCreatorFacetIKit = harden(
+  M.interface('IBCTokenCreatorFacetKit', {
+    // XXX refactor M.any()
+    getTokenMint: M.call().returns(M.any()),
+    getTokenIssuer: M.call().returns(M.any()),
+    getTokenBrand: M.call().returns(M.any()),
+    getTokenKit: M.call().returns(M.any()),
+  }),
+);
+
+/**
+ * @param {ZCF<IBCTokenTerms>} zcf
+ * @param {Record<string, unknown>} _privateArgs
+ * @param {import('@agoric/vat-data').Baggage} baggage
+ */
+export const prepare = async (zcf, _privateArgs, baggage) => {
```

Comment:
Have you seen mintHolder.js in agoric-sdk ? Does the same thing, afaict

Reply:
I saw it but did not look closely enough so thank you for the pointer.

I suppose we also do not need to produce an installation or instance - just a `myTokenKit` that holds the creator/mint facet.

Reply:
using `mintHolder.js` does involve starting a new instance of it